modules/data-catalog-tag/variables.tf

```diff
@@ -31,7 +31,12 @@ variable "tags" {
     column     = optional(string)
     location   = string
     template   = string
-    fields     = map(string)
+    fields = map(object({
+      double_value    = optional(number)
```

Comment:
Do we add `display_name` here then?

Reply:
Display name is an output of the resource. Removed in the main.tf, no need to add it here.